### DISCUSSION TITLE
Align contents in help-intro.pd for consistent layout

### DISCUSSION THREAD

**Discussion by ben-wes** [pure-data/pddp#176] [Ideas]
This is definitely a rather irrelevant topic and a minor issue. But since I've opened the `help-intro.pd` file through _right-click on canvas -> help_ quite often in the past 4 years, the "pain" starts to add up ...

The objects there are not consistently aligned and have different distances between them. This is certainly not an issue when it comes to usability. But since my original background is in graphic design, it always bothered me a bit. :)

I would offer to work through the whole file if that's a welcome contribution. There are a few other little things I'd also suggest in this context:
* header similar to help files with the "canvas line" and headline text
* use underscore-lines for "chapters" since the dashed lines are visually less calm
* summarize the part for Pd messages and GUI objects (since they both use additional patches for further reading) and have the same `<=` arrow there as for the reference clicks in all object help files 
* remove the `-` for the object description since imho this only adds visual elements that don't transport any information or help readability

I quickly applied this here for the upper part in comparison with the current version and with a variant of right-aligned objects. I'm not sure about right-aligning objects though - it helps to connect the object to the describing text, but in turn takes a bit from the readability in the object column:

![Screenshot 2023-11-29 at 13 15 27](https://github.com/pure-data/pddp/assets/1923787/d060c4bc-172a-4830-bd7d-30b25f3195d2)

**Comment by porres** [comment]
yeah, I'd keep '-' as it is used all throughout the documentation anyway, and sometimes you have more than one line.

>  I'm not sure about right-aligning objects though 

didnt really like it either

but hey, send some options, post them here as zipped files, let's check them

**Comment by ben-wes** [comment]
thanks for the feedback, @porres 

i made two variants now (with and without dashes) for consistent spacing and alignment everywhere:
* [help-intro-variants.zip](https://github.com/pure-data/pddp/files/13511827/help-intro-variants.zip)

i'm open to leaving out all the other changes here, of course - but would be happy about the layout alignments. :)

other changes were:
* header arrangement and changes in the header text
* changed `switch~` message to comment since i found the message quite confusing there without explanation
* underscore instead of dashes for lines (i like this version - but that's probably a matter of taste)
* "footer" line

what would be really nice in general, but off-topic here: if Pd could automatically scroll to the window area of a search result since i sometimes searched for stuff in this file and needed to watch out for the blue highlighting while scrolling through ...

**Comment by porres** [comment]
@ben-wes this could be an issue... I forgot about it, closing this discussion, we can sort this via chat as well
